dask_jobqueue/sge.py

```diff
@@ -58,7 +58,8 @@ def __init__(self, queue=None, project=None, resource_spec=None, walltime=None,
 
         super(SGECluster, self).__init__(**kwargs)
 
-        header_lines = ['#!/usr/bin/env bash']
+        header_lines = ['#!/bin/bash']
```

Comment:
can you change this back to `#!/usr/bin/env bash`? I must have missed this in the merge conflict resolution.